use-cases/predictive_maintenance/3_train_tune_predict_predmaint.ipynb

```diff
@@ -4,43 +4,58 @@
    "cell_type": "markdown",
```

Comment:
<p>Grammar (understood this is probably copy/paste from existing text) - Follow<code>Let's start by</code> with a list of actions ending in <code>-ing</code> (ie <code>Install</code> -&gt;  <code>Installing</code>,  <code>Instantiate</code> -&gt; <code>Instantiating</code> )  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done, for other notebooks as well</p>